Sources/SwiftParser/Declarations.swift

```diff
@@ -210,81 +210,99 @@ extension Parser {
     // while recovering to the declaration start.
     let recoveryPrecedence = inMemberDeclList ? TokenPrecedence.closingBrace : nil
 
-    switch self.canRecoverTo(anyIn: DeclarationKeyword.self, overrideRecoveryPrecedence: recoveryPrecedence) {
-    case (.import, let handle)?:
+    CanRecoverToSwitch: switch self.canRecoverTo(anyIn: DeclarationKeyword.self, overrideRecoveryPrecedence: recoveryPrecedence) {
```

Comment:
I don’t actually think we need the label. If we `break` in the nested `switch`, then it just executes to the end of the outer `switch`, which is the same as `break CanRecoverToSwitch`.

---

If we do need it: We currently all caps our labels, so for consistency, this should be `CAN_REVOVER_TO_SWITCH`.